src/aiida_workgraph/config.py

```diff
@@ -11,7 +11,7 @@
 
 def load_config() -> dict:
     """Load the configuration from the config file."""
-    config_file_path = AIIDA_CONFIG_FOLDER / "workgraph.json"
+    config_file_path = AiiDAConfigDir.get() / "workgraph.json"
```

Comment:
Try public API maybe? as mentioned in https://github.com/aiidateam/aiida-core/issues/6656#issuecomment-2531063738

BTW, is the workgraph.json config bind to a config or it is better to bind to a profile? @superstar54

Reply:
Thanks for pointing out this. For the moment, it's a global setting for all. But maybe in the future, we could also support overriding it for each profile.